test/pred-sets/ref/ccb_implicit_and_explicit_interactions.inv

```diff
@@ -1,5 +1,5 @@
 Version 8.11.0
-Id 
+Id
```

Comment:
also applies to this space, unless its trimmed when parsing it could shift everything as we read.

Reply:
Same answer as before, these files cannot be read